include/caliper/cali_macros.h

```diff
@@ -18,14 +18,20 @@
 
 #include "Annotation.h"
 
+// Macros for building variable names with other macros
+// These macros were obtained from:
+// https://stackoverflow.com/a/71899854
+#define CONCAT_(prefix, suffix) perfix##suffix
+#define CREATE_VAR_NAME(prefix, suffix) CONCAT_(prefix, suffix)
```

Comment:
I'd prefix these with `CALI_` just to be sure

Reply:
[d8055c6](https://github.com/LLNL/Caliper/pull/596/commits/d8055c60473c23fa29d5ad0332045540a7f8e9dd) resolves this and fixes unit tests.